pkg/proto/configuration/scheduler/scheduler.proto

```diff
@@ -294,3 +299,20 @@ message InitialSizeClassFeedbackDrivenAnalyzerConfiguration {
   // Recommended value: 32
   int32 history_size = 6;
 }
+
+message OverridingSizeClassAnalyzerConfiguration {
+  // The SizeClassAnalyzer to use when the specific platform key named
+  // below is not present.  If this is not provided, every action must
+  // specify the size class to use in its platform properties.
+  InitialSizeClassAnalyzerConfiguration base = 1;
```

Comment:
I think it's always the case you want to place PlatformOverridingSizeClassAnalyzer outside of FeedbackDrivenAnalyzer. Should we maybe get rid of the `oneof` above? As in, allow both fields to be set, but let 'overriding' wrap 'feedback_driven'.